.editorconfig

```diff
@@ -9,10 +9,11 @@ insert_final_newline = true
 end_of_line = lf
 
 [*.{kt, kts}]
+ktlint_code_style = intellij_idea
 ij_kotlin_imports_layout = *,java.**,javax.**,kotlin.**,kotlinx.**,^
 ij_kotlin_allow_trailing_comma = false
 ij_kotlin_allow_trailing_comma_on_call_site = false
 
 ktlint_standard_argument-list-wrapping = disabled
 ktlint_standard_value-argument-comment = disabled
-ktlint_function_naming_ignore_when_annotated_with = Composable
\ No newline at end of file
+ktlint_function_naming_ignore_when_annotated_with = Composable
```

Comment:
windows editing?

Reply:
No, just editing, IDE adds it automatically.

Note: all files (even .java-version) in the project have new line at end of file, why not .editorconfig?